src/iosrtcPlugin.swift

```diff
@@ -17,13 +17,19 @@ class iosrtcPlugin : CDVPlugin {
 	// PluginMediaStreamRenderer dictionary.
 	var pluginMediaStreamRenderers: [Int : PluginMediaStreamRenderer] = [:]
 	// Dispatch queue for serial operations.
-	let queue = dispatch_queue_create("cordova-plugin-iosrtc", DISPATCH_QUEUE_SERIAL)
+	var queue = dispatch_queue_create("cordova-plugin-iosrtc", DISPATCH_QUEUE_SERIAL)
```

Comment:
can this be defined somehow else not to repeat the code below? same goes for all other fields initialized inside pluginInitialize.